js/apps/events-page/models/requestModel.js

```diff
@@ -0,0 +1,30 @@
+((app) => {
+	'use strict';
+
+	const RequestModel = ($window, CONSTANTS) => {
+		return (requestModel) => {
+			const model = requestModel || {};
+
+			const startDateLocaleString = $window.moment().format();
+			const endDateLocaleString = $window.moment().add(30, 'd').format();
+			const eventTypes = model.eventTypes || [];
+
+			return {
+				StartDate: model.StartDate || startDateLocaleString,
+				EndDate: model.EndDate || endDateLocaleString,
+				Page: model.Page || 1,
+				IsOngoingVisible: model.IsOngoingVisible || true,
+				IsSpacesReservationVisible: model.IsSpacesReservationVisible || false,
+				Limit: model.limit || CONSTANTS.requestChunkSize,
+				EventsTypes: eventTypes, // HACK: API Needs this
```

Comment:
Not going ot block, but curious why this isn't `model.eventTypes || []`.

Reply:
good point, when i did that, i had an alias property. let me roll that back.